### ISSUE TITLE
false positive for 'wider' nested th on table 

### ISSUE THREAD

**Issue by gypsydave5** [squizlabs/HTML_CodeSniffer#285]
I think the following is a false positive. Given the following table:

<table>
<thead>
  <tr><th id="one">one</th><th id="two">two</th></tr>
</thead>
<tbody> 
<tr><th colspan="2" id="a">a</th></tr>
  <tr>
    <td headers="a one">a one</td>
    <td headers="a two">a two</td>
  </tr>
</tbody>

<tbody>
  <tr><th colspan="2" id="b">b</th></tr>
  <tr>
    <td headers="b one">b one</td>
    <td headers="b two">b two</td>
  </tr>
</tbody>
</table>

```html
<table>
<thead>
  <tr><th id="one">one</th><th id="two">two</th></tr>
</thead>
<tbody>
  <tr><th colspan="2" id="a">a</th></tr>
  <tr>
    <td headers="a one">a one</td>
    <td headers="a two">a two</td>
  </tr>
</tbody>

<tbody>
  <tr><th colspan="2" id="b">b</th></tr>
  <tr>
    <td headers="b one">b one</td>
    <td headers="b two">b two</td>
  </tr>
</tbody>
</table>
```

returns the following errors:

```
Error: Incorrect headers attribute on this td element. Expected "a b one" but found "a one"
Error: Incorrect headers attribute on this td element. Expected "a b two" but found "a two"
Error: Incorrect headers attribute on this td element. Expected "a b one" but found "b one"
Error: Incorrect headers attribute on this td element. Expected "a b two" but found "b two"
```

These errors appear both on the [HTML CodeSniffer website](http://squizlabs.github.io/HTML_CodeSniffer/) and when using [pa11y](https://pa11y.org/), which has a dependency on the HTML CodeSniffer.

[This example tutorial from WAI on W3](https://www.w3.org/WAI/tutorials/tables/examples/threeheaders/) also fails.

## Expectation

I'd expect there to be no error at all here; the tds are clearly related to only one of the headers above them, this is explicit both visually and in the markup.

**Comment by ironikart** [comment]
This is addressed with #290 and errors will no longer be shown for these types of tables.
